scripts/globals/mobskills/petrifactive_breath.lua

```diff
@@ -0,0 +1,25 @@
+---------------------------------------------
+-- Petrifactive Breath
+--
+-- Description: Petrifies a single target.
+-- Type: Breath
+-- Utsusemi/Blink absorb: Ignores shadows
+-- Range: Unknown
+-- Notes:
+---------------------------------------------
+require("scripts/globals/monstertpmoves")
+require("scripts/globals/settings")
+require("scripts/globals/status")
+---------------------------------------------
+
+function onMobSkillCheck(target,mob,skill)
+    return 0
+end
+
+function onMobWeaponSkill(target, mob, skill)
+    local typeEffect = dsp.effect.PETRIFICATION
```

Comment:
same here, can skip the var

Reply:
This one though... if you can't figure out what this is without the var then... idk

I will get rid of those in the future. 👍